### PR TITLE
quality-standard: show rule popularity

### PR DESCRIPTION
**Pull request by leplatrem**
Screenshot 2023-10-19 at 13 00 19

### CODE REVIEW

File: docs/quality-standards/scorecards.yaml [hunk 1]
```diff
@@ -701,7 +701,7 @@ Telescope:
       deployment-integration-test:
         compliant: No
       deployment-auto-pull-requests:
-        compliant: Partially
+        compliant: No
```
Comment (leplatrem) [inline docs/quality-standards/scorecards.yaml:704]: driveby: we don't deploy telescope on pull-requests

I assume that if a rule has no popularity at all (like this one, we could choose to drop it)

**Review by grahamalama — APPROVED** [review]
